blockchain/vm/evm.go

```diff
@@ -538,6 +538,31 @@ func (evm *EVM) CreateWithAddress(caller types.ContractRef, code []byte, gas uin
 	return evm.create(caller, codeAndHash, gas, value, contractAddr, humanReadable, codeFormat)
 }
 
+func (evm *EVM) CurrentCodeFormat() params.CodeFormat {
+	switch {
+	case evm.chainRules.IsIstanbul:
+		return params.CodeFormatEVMIstanbulCompatible
+	default:
+		return params.CodeFormatEVM
+	}
+}
+
+func (evm *EVM) GetPrecompiledContractMap(caller, address common.Address) map[common.Address]PrecompiledContract {
```

Comment:
The second parameter `address` is not needed anymore.

Reply:
thanks, reflected at https://github.com/klaytn/klaytn/pull/912/commits/94cfe30209ae84db4dc9fdfdbb376499c2de156c